Added Inventory Item Model and Service

deepsidhu1313 opened the pull request:
* Added Model and Service for Inventory Item (https://shopify.dev/api/admin-rest/2022-04/resources/inventoryitem).
* Added Unit test for the new service

Wryte commented on src/Models/InventoryItem.php:
```diff
@@ -0,0 +1,221 @@
+
+    /** @var string */
+    protected $cost;
+    /** @var string|null */
```
super nit: spacing inconsistent here

deepsidhu1313 replied:
is this better ?

Wryte reviewed (changes requested):
Are the nullable properties on the model from the documentation? For the setters of those properties should they be able to take null as well?

deepsidhu1313 commented:
removed type hinting and updated code to accept null values in setters. I took the example given on shopify https://shopify.dev/api/admin-rest/2022-04/resources/inventoryitem to create this model and service
